corehq/apps/domain/templates/domain/confirm_plan.html

```diff
@@ -1,8 +1,20 @@
 {% extends "domain/base_change_plan.html" %}
 {% load i18n %}
 {% load hq_shared_tags %}
+{% requirejs_main 'accounting/js/confirm_plan' %}
 
 {% block form_content %}
+{% initial_page_data 'is_upgrade' is_upgrade %}
+{% initial_page_data 'next_invoice_date' next_invoice_date %}
+{% initial_page_data 'current_plan' current_plan %}
+{% initial_page_data 'new_plan_edition' new_plan_edition %}
+{% initial_page_data 'new_plan_name' plan.name %}
+{% initial_page_data 'is_downgrade_before_minimum' is_downgrade_before_minimum %}
+{% initial_page_data 'current_subscription_end_date' current_subscription_end_date %}
+{% initial_page_data 'start_date_after_minimum_subscription' start_date_after_minimum_subscription %}
+{% registerurl 'email_on_downgrade' domain %}
+{% registerurl 'confirm_billing_account_info' domain %}
+<div class="container" id="hq-content">
```

Comment:
This should have a different id - there's already an `#hq-content` on this page, defined in `hqwebapp/two_column.html` (earlier comment wasn't clear on that, sorry)